app/jobs/analytics_event_job.rb

```diff
@@ -0,0 +1,13 @@
+class AnalyticsEventJob < ActiveJob::Base
+  queue_as :analytics
+
+  TRACKER = if Rails.env.test?
+              Staccato.tracker(nil)
+            else
+              Staccato.tracker('UA-48605964-44', nil, ssl: true)
```

Comment:
I think PK was right, even though this is not expected to change, this is still a configurable option and ought to be pulled to an ENV var.